features/step_definitions/matching.js

```diff
@@ -113,20 +123,23 @@ module.exports = function () {
                     var testSubMatching = (sub, si, scb) => {
                         if (si >= subMatchings.length) {
                             ok = false;
-                            q.abort();
+                            // q.abort();
                             scb();
```

Comment:
I'm removing this if check because we do this test before entering the testSubMatching function! I just didn't realize that before!